src/App.js

```diff
@@ -1,11 +1,38 @@
-import React from 'react';
+import axios from 'axios';
+import React, { useEffect, useState } from 'react';
 import './App.css';
+import PokemonCard from "./PokemonCard";
+import pokemon_logo from "./assets/International_Pokémon_logo.svg"
 
 function App() {
+    const [pokenames, setPokenames] = useState(null);
+    useEffect(() => {
+        console.log('fetch names here');
+
+        async function fetchNames(){
+            const response = await axios.get('https://pokeapi.co/api/v2/pokemon?limit=20');
+            console.log("de hek",response.data.results);
+            setPokenames(response.data.results);
+        };
+        fetchNames();
+
+
+    }, [])
+
+
   return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
+      <div className='container'>
+          <img className="logo" src={pokemon_logo} alt="pokemon logo"/>
+          {pokenames ? (
```

Comment:
Ik zou deze check expliciet doen met een useState `loading`. Die wordt dan op loading gezet voordat het request plaatsvindt en op false gezet zodra het resultaat opgehaald is